src/offline/utils.ts

```diff
@@ -61,4 +62,5 @@ export {
     deriveXAddress,
     signPaymentChannelClaim,
     verifyPaymentChannelClaim,
+    Wallet,
```

Comment:
This should be exported at the top level instead of `utils`.

Reply:
Wallet is indirectly exported at the top level since `offline/utils` is exported at the top level.

Reply:
for version 2, it should be explicitly exported at the toplevel. i don't think that `offline` should exist, and given that this isn't a `util`, I don't think it belongs here in the first place. IMO it should be either `src/Wallet.js` or `src/Wallet/index.js`

Reply:
I moved it under `src/Wallet.js` in this updated PR.